invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -22,20 +22,60 @@
           "type": "keyword"
         },
         "$schema": {
-          "type": "keyword"
+          "type": "text",
+          "index": false
         },
         "recid": {
-          "type": "long"
+          "type": "keyword"
         },
         "title": {
-          "type": "text"
+          "type": "text",
+          "copy_to": "suggest_title"
+        },
+        "suggest_title": {
+          "type": "completion"
         },
```

Comment:
Sort: shall we add it to a ticket along with the other sorting issue above?

Suggest_title: Its used for autocompletion https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#completion-suggester (I prefer n-grams since they provide better results. But this are much more efficient)

Reply:
For "title" sorting, we might as well just add it now and save us the overhead of a ticket.

For `suggest_title`, I was actually wondering about the use-case. Where/when are we auto-completing titles?

Reply:
For sorting. It works for me with React Search kit. AFAIK, we are intending to use it, but we wont be sure 100% until January when we start with UI.

Suggestion, I fear I dont see it in Zenodo for titles. I see it for licenses. In other instances (e.g. CDS videos its used already, see screenshot). I think is a nice improvement, shall we leave it?
![image](https://user-images.githubusercontent.com/6756943/67189778-21b8c000-f3ef-11e9-9c61-f712f8e082bc.png)